tests/Oryx.BuildImage.Tests/Ruby/RubySampleAppsTest.cs

```diff
@@ -119,5 +119,38 @@ public void Builds_JekyllStaticWebApp_When_Apptype_Is_SetAs_StaticSiteApplicatio
                 CommandArguments = new[] { "-c", script }
             });
         }
+
+        [Theory]
+        [InlineData("vso-focal")]
+        [InlineData("azfunc-jamstack")]
+        public void Builds_JekyllStaticWebApp_UsingCustomBuildCommand(string buildImage)
+        {
+            // Arrange
+            var appName = "Jekyll-app";
+            var volume = CreateSampleAppVolume(appName);
+            var appDir = volume.ContainerDir;
+            var appOutputDir = "/tmp/app-output";
+            var script = new ShellScriptBuilder()
+                .AddDefaultTestEnvironmentVariables()
+                .SetEnvironmentVariable("CUSTOM_BUILD_COMMAND", "touch example.txt")
```

Comment:
ah .. ok .. so it's not really a property or argument that we pass in oryx command line .. it's just and environment variable